packages/codemod/src/utils/prop-utils.ts

```diff
@@ -79,12 +81,37 @@ function getPropValue(j: JSCodeshift, prop: JSXAttribute): undefined | boolean |
       // e.g. <Button text={true} /> or <Button text={"text"} />
       return value.expression.value;
     }
-  }
 
+    if (value.expression.type === "MemberExpression") {
+      // e.g. <Button border={Box.borders.DEFAULT} />
+      return j(value.expression).toSource();
+    }
+  }
   // can be very complex, we'll have to compare strings
   return value ? j(value).toSource() : undefined;
 }
 
+export function setPropValue(
+  j: JSCodeshift,
+  attributePath: ASTPath<JSXAttribute>,
+  newValue: {
+    value: string | number | boolean;
+    type: typeof memberExpression | typeof literal;
+  }
+): void {
+  if (typeof newValue.value !== "string") {
+    const newValueIsTrue = typeof newValue.value === "boolean" && newValue.value;
+    attributePath.node.value = newValueIsTrue ? null : j.jsxExpressionContainer(j.literal(newValue.value));
+  } else {
+    if (newValue.type === memberExpression) {
+      const objectValue = j(`${newValue.value}`).find(j.ExpressionStatement).get().node.expression;
+      attributePath.node.value = j.jsxExpressionContainer(objectValue);
```

Comment:
might worth to add a protection
something like 
```suggestion
      const objectValue = j(`${newValue.value}`).find(j.ExpressionStatement).get()?.node?.expression;
      attributePath.node.value = j.jsxExpressionContainer(objectValue);
```
or
```suggestion
      const expression = j(`${newValue.value}`).find(j.ExpressionStatement);
      if (!expression.get()) return;
      attributePath.node.value = j.jsxExpressionContainer(expression.get().node.expression);
```

Reply:
Added!